docs/api.rst

```diff
@@ -427,6 +427,66 @@ Possible Responses
 | 500 Server Error | Internal Raiden node error|
 +------------------+---------------------------+
 
+Transfers
+=========
+
+Initiating a Transfer
+---------------------
+
+You can create a new transfer by making a ``PUSH`` to the following endpoint along with a json payload containing
+the transfer details such as amount and identifier. Identifier is optional.
+
+``PUSH /api/<version>/transfers/<token_address>/<target_address>``
+
+Example Request
```

Comment:
As I said in the other PR, we should add curl examples to all of the endpoints. It's a good idea but I would like to do it all in one swoop as 1 commit and it's not part of this PR.